nautobot/ipam/factory.py

```diff
@@ -158,7 +158,7 @@ class Meta:
         )
 
     # note that name is *not* globally unique
-    name = factory.Faker("color_name")
+    name = factory.LazyFunction(lambda: f"{faker.Faker().color_name()}{faker.Faker().pyint()}")
```

Comment:
Would this make sense to be a [LazyAttributeSequence](https://factoryboy.readthedocs.io/en/stable/reference.html#lazyattributesequence)?

Reply:
Could do, then the names would be (random color + sequential number) instead of (random color + random number). I don't feel strongly about one versus the other, I just needed to add something because color names alone weren't proving to be unambiguous enough.

Reply:
My thought was this line would be easier to read as:
```py
name = factory.LazyAttributeSequence(lambda n: f"{faker.Faker().color_name()}{n}")
```

Reply:
Easier to read but guarantees that names are globally unique, which isn't actually a requirement of the model.